### DISCUSSION TITLE
Is it compatible with VueJS 2.7?

### DISCUSSION THREAD

**Discussion by MartinX3** [nuxt/nuxt#10552] [Questions (Nuxt 2)] [answered]
Or do we need to wait for a new NuxtJS 2.x release?

https://github.com/vuejs/vue/releases/tag/v2.7.0

Edit:
After looking through the commits I see that it got added and probably will probably be in the next release.

Is there a NuxtJS 2.x roadmap?

**Comment by cannap** [comment, marked as answer]
yea they work on 

https://twitter.com/Atinux/status/1532405926405316608?s=20&t=XINq5s-Pf6hGWGb3ivBiiA

**Comment by MartinX3** [comment]
https://github.com/nuxt/bridge uses Vue 2.7.
But sadly it's static site generation is broken.
```
    ssr: false,
    target: 'static',
```

**Reply by joaopedrodcf** [reply]
If I use nuxt brigde , SSG and vue 2.7 SSG will fail ? Can you give me the URL of that specific issue ?
